### PR TITLE
Secure file host (Desktop connection)

### LINKED ISSUE #91: Connect through desktop application (local vault files)
Option to connect to archives available in the desktop application.

Once an encrypted channel is set up, access to the filesystem can be provided. Files read and written should be limited to `.bcup` files only.

### PR DESCRIPTION
**Pull request by perry-mitchell**
Closes #91.

Relates to buttercup/buttercup-desktop#692

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,5 +1,12 @@
 {
-  "presets": ["env", "react"],
+  "presets": [
+    ["env", {
+      "targets": {
+        "chrome": 60
```
Comment (sallar) [inline .babelrc]: do you think chrome is enough for this env? I imagine firefox supports the same stuff right?

File: package.json [hunk 2]
```diff
@@ -75,6 +75,7 @@
     "history": "^4.7.2",
     "html-webpack-plugin": "^2.30.1",
     "husky": "^0.14.3",
+    "iocane": "^1.0.2",
```
Comment (sallar) [inline package.json]: Do we need this anymore? because you are now using iocane from Buttercup right?

File: .babelrc (hunk 1 above)
Comment (perry-mitchell) [reply .babelrc]: I'll add FF too :)

File: package.json (hunk 2 above)
Comment (perry-mitchell) [reply package.json]: Removed! 😃

**Comment by aleclarson** [comment]
I've enabled "Secure file host" in the desktop app. How do I make the Firefox browser extension use it?

**Pull request merged**